docs/monitoring/spm-faq.md

```diff
@@ -731,6 +731,26 @@ After that, you should be able to add the key to your system.
 wget -O - https://pub-repo.sematext.com/debian/sematext.gpg.key | sudo apt-key add -
 ```
 
+### How to prevent "Pending" status when installing Sematext Agent on a Kubernetes Cluster?
+
+If you notice that the pods for Sematext Agent are stuck in a "Pending" status with an error similar to "Port is already in use" during the installation on your Kubernetes Cluster, it may related to `hostNetwork` setting. 
+
+Sematext Agent requires `hostNetwork` access in order to access and monitor the Kubernetes control plane components. However, if you enable `hostNetwork` and the default port (8675) is already being used on your host, it can result in the Sematext Agent pods remaining in a "Pending" state. To resolve this problem, you need to modify the port being used.
+
+If you have concerns about `hostNetwork` usage or you don't want to use it in your system, please refer to [this page](https://sematext.com/docs/agents/sematext-agent/kubernetes/hostnetwork/).
+
+If you will continue to use `hostNetwork` (which is required for fully featured Kubernetes monitoring), please follow the steps below, depending on the installation method you are using:
+
+**Helm:**
+
+Include the following parameter in your regular helm installation command: `--set agent.service.port=<Free-Port>`. Replace `<Free-Port>` with an available port number that is not already in use.
```

Comment:
Replace `<Free-Port>` with an available port number that is not already in use. ==> Replace `<Free-Port>` with a port that is not in use.